cg/services/run_devices/pacbio/post_processing_service.py

```diff
@@ -49,7 +50,10 @@ def __init__(
         to_raise=PostProcessingError,
     )
     def post_process(self, run_name: str, dry_run: bool = False) -> None:
-        LOG.info(f"Starting PacBio post-processing for run: {run_name}")
+        if self._is_run_post_processed(run_name):
```

Comment:
Did we not say that this should check should be where we pick up the runs to iterate over?

Reply:
Good catch, I forgot about that. I also double checked and if we try to post-process an already existing smrt cell it will fail before writing anything into the database or saving stuff to Housekeeper so this is not needed here

Reply:
fixed